### PR TITLE
Add arm swing packet translator

### PR DESCRIPTION
**Pull request by johnnywoof**
This will make it so that java clients will see arm swings from PE clients.

### CODE REVIEW

File: src/main/java/org/dragonet/proxy/network/PacketTranslatorRegister.java [hunk 1]
```diff
@@ -124,6 +124,9 @@
         PE_TO_PC_TRANSLATOR.put(MobEquipmentPacket.class, new PEPlayerEquipmentPacketTranslator());
         PE_TO_PC_TRANSLATOR.put(InventoryTransactionPacket.class, new PEInventoryTransactionPacketTranslator());
         PE_TO_PC_TRANSLATOR.put(BlockPickRequestPacket.class, new PEBlockPickRequestPacketTranslator());
+
+        // Player
+		PE_TO_PC_TRANSLATOR.put(AnimatePacket.class, new PEAnimatePacketTranslator());
```
Comment (DefinitlyEvil) [inline src/main/java/org/dragonet/proxy/network/PacketTranslatorRegister.java]: Extra spaces xD

File: src/main/java/org/dragonet/proxy/network/translator/pe/PEAnimatePacketTranslator.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+public class PEAnimatePacketTranslator implements IPEPacketTranslator<AnimatePacket> {
+
+	@Override
+	public Packet[] translate(UpstreamSession session, AnimatePacket packet) {
+
+		switch (packet.action) {
+			case AnimatePacket.ANIMATION_SWING_ARM:
+				return new Packet[]{new ClientPlayerSwingArmPacket(Hand.MAIN_HAND)};
+			default:
+				return new Packet[]{};
```
Comment (DefinitlyEvil) [inline src/main/java/org/dragonet/proxy/network/translator/pe/PEAnimatePacketTranslator.java]: You can just return null. ;)

**Review by DefinitlyEvil — CHANGES_REQUESTED** [review]

**Review by DefinitlyEvil — APPROVED** [review]

**Review by HoverEpic — CHANGES_REQUESTED** [review]
Same here, \t formating is not good, use 4 spaces.

**Comment by johnnywoof** [comment]
My bad, sorry about that. It should use spaces now

**Review by HoverEpic — APPROVED** [review]

**Pull request merged**